### ISSUE TITLE
Possible README typo

### ISSUE THREAD

**Issue by AustinGil** [unjs/unenv#167]
Should this line in the README say `node` or `nodeless`?

https://github.com/unjs/unenv/blob/a09cedd6f8f48a35643ab6a690a1b4e83cfadfe3/README.md?plain=1#L44

The full snippet is
```js
import { env, nodeless } from "unenv";

const envConfig = env(node, {});
```

**Comment by manniL** [comment]
It should be

```js
import { env, node } from "unenv";

const envConfig = env(node, {});
```

PR welcome

**Comment by pi0** [comment]
fixed by #172
